### PR TITLE
Fix typos and ignore byte-compiled cached files

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S po -L filetest,keypair`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -352,7 +352,7 @@
 * Support legacy devices without a Name property
 * (Covered) Crash in agent's passkey / pin code methods
 * Fix fading in GtkAnimation TreeRow(Color)Fade and CellFade classes.
-* Do not explixitely destoy DeviceSelectorDialog blueman-sendto.
+* Do not explixitely destroy DeviceSelectorDialog blueman-sendto.
```
Comment (cschramm) [inline CHANGELOG.md:355]: Well... 😄

File: CHANGELOG.md [hunk 2]
```diff
@@ -421,7 +421,7 @@ See https://github.com/blueman-project/blueman/wiki/PolicyKit
 
 * Listing rfcomm devices was broken
 * Serial devices where not properly disconnected
-* Close confirmation dialog on cancelation
+* Close confirmation dialog on cancellation
```
Comment (cschramm) [inline CHANGELOG.md:424]: It's correct American English.

File: stubs/gi/repository/GLib.pyi [hunk 3]
```diff
@@ -2450,7 +2450,7 @@ class ErrorType(Enum, builtins.int):
 
 
 class FileError(Enum, builtins.int):
-    ACCES = ...  # type: FileError
+    ACCESS = ...  # type: FileError
```
Comment (cschramm) [inline stubs/gi/repository/GLib.pyi:2453]: That's external API and not supposed to be a complete word.

File: stubs/gi/repository/GLib.pyi [hunk 4]
```diff
@@ -2649,7 +2649,7 @@ class SliceConfig(Enum, builtins.int):
 
 
 class SpawnError(Enum, builtins.int):
-    ACCES = ...  # type: SpawnError
+    ACCESS = ...  # type: SpawnError
```
Comment (cschramm) [inline stubs/gi/repository/GLib.pyi:2652]: See above.

**Review by cschramm — CHANGES_REQUESTED** [review]
Thank you, but this will need some tuning.

File: CHANGELOG.md (hunk 1 above)
Comment (infirit) [reply CHANGELOG.md:355]: Weird it got the destoy but not `explixitely ` -> `explicitly`. @kianmeng can you add this change?

**Review by cschramm — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kianmeng) [reply CHANGELOG.md]: @infirit FYI, https://github.com/codespell-project/codespell/pull/2691

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

File: CHANGELOG.md (hunk 1 above)
Comment (infirit) [reply CHANGELOG.md]: Nice 👍

**Pull request merged**